skimage/registration/_optical_flow.py

```diff
@@ -11,15 +11,15 @@
 from ._optical_flow_utils import coarse_to_fine
 
 
-def _tvl1(image0, image1, flow0, attachment, tightness, nwarp, niter,
-          tol, prefilter):
+def _tvl1(reference_image, moving_image, flow0, attachment, tightness, nwarp,
+          niter, tol, prefilter):
```

Comment:
Referring to your [comment ](https://github.com/scikit-image/scikit-image/issues/4154#issuecomment-531064682) on #4154, I suggest to replace `niter` by `num_iter` before releasing v0.16 to anticipate ;-)